tables/ref_isolate_pairs/stuart22-pair.csv

```diff
@@ -0,0 +1,4 @@
+ref_name,control_iso_name,iso_name
+Stuart22,Wuhan-Hu-1,Wuhan-Hu-1 Spike
```

Comment:
@zachandfox I think the 2nd line is an error. Two reasones.

1. Wuhan-Hu-1 and Wuhan-Hu-1 Spike are the same, they don't have mutaions
2. Wuhan-Hu-1 is test on virus isolates and Wuhan-Hu-1 spike is test on pseudovirus, they can't be paired.